src/interfaces/registry.ts

```diff
@@ -19,6 +19,7 @@ interface ApplicantData {
 
 interface Challenge extends RegistryChallenge {
   challengeID: string
```

Comment:
changed to `challenge` in the other structs (comp, .js)

Reply:
But this is the `Challenge`. Feels weird for `Challenge` to have a `challenge` attribute.